galsim/chromatic.py

```diff
@@ -142,6 +142,25 @@ def resize_multiplier_cache(maxsize):
         """
         ChromaticObject._multiplier_cache.resize(maxsize)
 
+    def _fiducial_profile(self, bandpass):
+        """
+        Return a fiducial achromatic profile of a chromatic object that can be used to estimate
+        default output image characteristics, or in the case of separable profiles, can be scaled to
+        give the monochromatic profile at any wavelength or the wavelength-integrated profile.
+        """
+        bpwave = bandpass.effective_wavelength
+        candidate_waves = [bpwave, 0.5 * (bandpass.blue_limit + bandpass.red_limit)]
+        candidate_waves = np.union1d(candidate_waves, bandpass.wave_list)
+        candidate_waves = np.union1d(candidate_waves, self.wave_list)
```

Comment:
>And another efficiency point. np.union1d includes a sort, which you don't care about, since you're resorting later. Better to just do candidate_waves += wave_list here as a concatenation.

It turned out to be easier to work with numpy.concatenate than +=, but point taken.  Fixed.